### PR TITLE
Add support for content lib problems in student response report.

### PR DESCRIPTION
**Pull request by dcadams**
Structure of content lib problems in Mongo being different to other problems requires code change.

### CODE REVIEW

**Comment by dcadams** [comment]
retest this please

**Comment by dcadams** [comment]
@caesar2164

File: lms/djangoapps/instructor_analytics/tests/test_basic.py [hunk 1]
```diff
@@ -573,39 +574,87 @@ def test_invalid_module_state(self):
...
+            parent_location=unit.location,
+            category='problem',
+            display_name=u'test problem',
+        )
+        submit_and_compare1 = ItemFactory.create(
+            parent_location=unit.location,
+            category='submit-and-compare',
+            display_name=u'test submit_and_compare1',
+        )
+        submit_and_compare2 = ItemFactory.create(
```
Comment (caesar2164) [inline lms/djangoapps/instructor_analytics/tests/test_basic.py]: having two items defined exactly the same, but named differently is weird

**Comment by caesar2164** [comment]
apart from my comment above, I'm 👍

File: lms/djangoapps/instructor_analytics/tests/test_basic.py (hunk 1 above)
Comment (caesar2164) [inline lms/djangoapps/instructor_analytics/tests/test_basic.py]: per F2F it NEEDS to be two different problem definitions, so I suggested giving them meaningful names.

Comment (dcadams) [inline lms/djangoapps/instructor_analytics/tests/test_basic.py]: Clarifying, this is caused by a database uniqueness constraint.

**Comment by dcadams** [comment]
retest this please

**Comment by dcadams** [comment]
retest this please

**Pull request merged**